manuscript/book.md

```diff
@@ -116,13 +116,13 @@ const names = ['Bilbo Baggins', 'Gandalf', 'Gollum'];
 const kebabNames = names.map(name => _.kebabCase(name));
 ```
 
-This version is much easier to read because we know that the `.map()` method transforms an array by keeping the same number of items. And unlike `.forEach()`, it doesn’t require a custom implementation nor mutate an output array. Also the callback function is now pure: it doesn’t access any variables in the parent function, only function arguments.
+This version is much easier to read because we know that the `.map()` method transforms an array by keeping the same number of items. And unlike `.forEach()`, it doesn’t require a custom implementation nor mutate an output array. Also, the callback function is now pure: it doesn’t access any variables in the parent function, only function arguments. Giving you the benefit of having to only reason about values and not about state
```

Comment:
⌛️ I don't understand this sentence, but maybe I can improve something here.

Reply:
```suggestion
This version is much easier to read because we know that the `.map()` method transforms an array by keeping the same number of items. And unlike `.forEach()`, it doesn’t require a custom implementation nor mutate an output array. Also, the callback function is now pure: it merely transforms input arguments to the output value without any side effects.
```

Still doesn't explain much — I need to think how and where to explain things like "what is a pure function".